### PR TITLE
flamenco, ci: Add vm_validate fixtures to CI

### PR DESCRIPTION
**Pull request by ravyu-jump**
- Adds vm_validate fixtures to CI tests
- Minor refactoring of the `vm_validate` fuzz harness to match Agave's behavior with `bytes` type

### CODE REVIEW

File: src/flamenco/runtime/tests/fd_exec_sol_compat.c [hunk 1]
```diff
@@ -335,6 +335,32 @@ sol_compat_syscall_fixture( fd_exec_instr_test_runner_t * runner,
   return ok;
 }
 
+int 
+sol_compat_validate_vm_fixture( fd_exec_instr_test_runner_t * runner,
+                                uchar const *                 in,
+                                ulong                         in_sz ) {
+  // Decode fixture
+  fd_exec_test_validate_vm_fixture_t fixture[1] = {0};
+  if ( !sol_compat_decode( &fixture, in, in_sz, &fd_exec_test_validate_vm_fixture_t_msg ) ) {
```
Comment (ripatel-fd) [inline src/flamenco/runtime/tests/fd_exec_sol_compat.c:344]: code style

File: contrib/test/run_test_vectors.sh [hunk 2]
```diff
@@ -40,4 +40,7 @@ cat contrib/test/elf-loader-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_
 LOG=$LOG_PATH/test_exec_instr
 cat contrib/test/instr-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_instr --log-path $LOG --log-level-stderr 4
 
+LOG=$LOG_PATH/test_vm_validate
+cat contrib/test/vm_validate-fixtures.list | xargs ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```
Comment (ripatel-fd) [inline contrib/test/run_test_vectors.sh:44]:
```suggestion
<contrib/test/vm_validate-fixtures.list xargs ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```

Comment (ripatel-fd) [reply contrib/test/run_test_vectors.sh:44]: or 
```suggestion
xargs -a contrib/test/vm_validate-fixtures.list ./$OBJDIR/unit-test/test_exec_sol_compat --log-path $LOG --log-level-stderr 4
```
(saves a pipe)

**Comment by ravyu-jump** [comment]
@ripatel-fd diffs https://github.com/firedancer-io/firedancer/compare/1d4144f61..b177eb293

**Review by ripatel-fd — APPROVED** [review]
Looks great!

**Pull request merged**